manifests/08-clusteroperator.yaml

```diff
@@ -3,3 +3,7 @@ kind: ClusterOperator
 metadata:
   name: image-registry
 spec: {}
+status:
+  versions:
+    - name: operator
+      version: "0.0.1-snapshot"
```

Comment:
this is the indication the CVO consumes to look for a matching level?

Reply:
This is replaced by the release image version - so for CI becomes `4.0.0-0.ci-XXXX` or `0.0.1-2019-blah-blah`.